tests/test_cli.py

```diff
@@ -315,3 +315,48 @@ def test_suggests_single_task_if_export_crashes(tmp_empty, monkeypatch):
 
 
 # adds import if needed / and doesn't add import pickle
+
+
+def test_clean_py():
```

Comment:
add the following argument to all your tests: `tmp_empty`. This will automatically move the test into a temporary directory, to prevent creating these files in the repository

these are called fixtures, you can learn about them here: https://docs.pytest.org/en/6.2.x/fixture.html